### PR TITLE
Add MDB Pi Hat from Qibixx

### PR DESCRIPTION
**Pull request by bileltechno**
It was testd using publish/unpublish.

### CODE REVIEW

**Comment by lurch** [comment]
The MarkDown file refers to both `Multi-Drop-Bus` **and** `Machine Data Bus` ?

**Comment by bileltechno** [comment]
Sorry I fixed the description.
It lust refers only the Machine Data Bus.

File: draft/overlay/mbd-pi-hat.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+---
+name: MDB HAT
+class: board
+type: io
+formfactor: HAT
+manufacturer: Qibixx
+description: MDB (Machine Data Bus) Bus Interface and Sniffer for the Raspberry Pi
+url: https://qibixx.com
+buy: https://qiba.pt/
+image: 'mbd-pi-hat.png'
```
Comment (lurch) [inline draft/overlay/mbd-pi-hat.md]: Would it make more sense if the image was called `mdb-pi-hat.png` ? :wink:

**Comment by Gadgetoid** [comment]
Thank you both! Will look into getting this merged ASAP.

File: draft/overlay/mbd-pi-hat.md (hunk 1 above)
Comment (Gadgetoid) [reply draft/overlay/mbd-pi-hat.md]: I had to look at this three times across two separate days before I realised what you meant :D I'll correct it when I merge.

**Pull request merged**